extension/package.json

```diff
@@ -1449,7 +1449,7 @@
       },
       {
         "view": "dvc.views.welcome",
-        "contents": "New to the extension?\n[Show Walkthrough](command:dvc.getStarted)\n\nThe extension is currently unable to initialize.\n[Show Setup](command:dvc.showExperimentsSetup)",
+        "contents": "New to the extension?\n[Show Walkthrough](command:dvc.getStarted)\n\nThe extension is currently unable to initialize.\n[Show Setup](command:dvc.showDvcSetup)",
```

Comment:
[Q] What do the "Show Experiments", "Show Plots" and "Run Experiment" buttons redirect to when the CLI is not setup? I think it should be the new section.

See `showSetupOrExecuteCommand`.